### DISCUSSION TITLE
Need to integrate sunbird players on our new react UI for learner flow

### DISCUSSION THREAD

**Discussion by paritshivani** [Sunbird-Ed/Community#737] [General Discussion]
Hi team,
Hope you are doing well!
Here we have came up with new approach of using sunbird ED. Where we are trying to build new UI for existing sunbird Ed portal features using react! this will be lighter version of current system.
We need help in integrating sunbird players to react app.
How can we do this?
Currently we are doing following steps to integrate it with react app
1. We have deployed players on saperate IP .
2. Then integrated it in iframe in our  react code and passed IP address, content data , user data to it
3. Now pdf, video and youtube contents are playing fine with this but we are facinf issue with assesment, html5 and epub typas of content.
4. There are some Player APIs i can see failing with 304 status code.
5. We have temporary solution to store this contents to saperate bucket and pass the bucket path to Player streaming url 

We wanted to know weather this is the correct way or not ?
- if yes why we are facing issue with these content types ?
- if No, How can we achieve this?

@vinukumar-vs ,@rajeevsathish, @AmiableAnil , @Krishnaj20, @chitranshu-keshav, @SanthoshVasabhaktula, @gandham-santhosh andham-santhosh, @coolbung @komalm, @Shakthieshwari

**Comment by Abhilash-Kishor** [comment]
@madhucr @pramodkvarma

I'm reaching out to you today to request your help with a ticket for NULP. We've been working on it for several days now and are encountering above challenges.

**Comment by HarishGangula** [comment]
@paritshivani Raise this issue in https://github.com/orgs/Sunbird-Knowlg/discussions

**Comment by HarishGangula** [comment]
As we haven't received a response from you, we will proceed to close this thread. If you have any further questions or require assistance in the future, please don't hesitate to start a new thread. Thank you.
